rest/data/project.go

```diff
@@ -108,12 +109,19 @@ func CreateProject(ctx context.Context, env evergreen.Environment, projectRef *m
 		}
 	}
 	if err := VerifyUniqueProject(projectRef.Id); err != nil {
-		return err
+		return false, err
 	}
 	// Always warn because created projects are never enabled.
-	_, err := model.ValidateProjectCreation(projectRef.Id, env.Settings(), projectRef)
+	catcher := grip.NewBasicCatcher()
```

Comment:
Can we call this warningCatcher or something?

Reply:
done